plugins/pom.xml

```diff
@@ -27,44 +27,30 @@ Copyright (c) 2016, Oracle and/or its affiliates. All rights reserved.
     <groupId>org.opensolaris.opengrok</groupId>
     <artifactId>plugins</artifactId>
     <packaging>jar</packaging>
-    <version>1.1-SNAPSHOT</version>
+    <version>1.1-rc22</version>
     <name>OpenGrok authorization plugins</name>
 
     <parent>
         <groupId>org.opensolaris.opengrok</groupId>
-        <artifactId>opengrok</artifactId>
-        <version>1.1-SNAPSHOT</version>
+        <artifactId>opengrok-top</artifactId>
+        <version>1.1-rc22</version>
     </parent>
 
     <build>
-        <sourceDirectory>.</sourceDirectory>
-	<!-- Note that the plugins/UserPlugin/test/ directory is added via the build-helper plugin below -->
-        <testSourceDirectory>LdapPlugin/test</testSourceDirectory>
+        <sourceDirectory>src</sourceDirectory>
```

Comment:
I think we should adhere to standard project structure, e.g. `src/main/java`.

Reply:
I think this change will be inevitable ... (look at gradle, sbt, ... )